source/contracts/reporting/DisputeBondToken.sol

```diff
@@ -32,13 +32,17 @@ contract DisputeBondToken is DelegationTarget, ITyped, Initializable, ERC20Basic
         return true;
     }
 
-    function withdraw() public returns (bool) {
+    function withdraw(bool forgoFees) public returns (bool) {
         require(msg.sender == bondHolder);
         bool _isFinalized = market.getReportingState() == IMarket.ReportingState.FINALIZED;
         require(!market.isContainerForDisputeBondToken(this) || (_isFinalized && market.getFinalPayoutDistributionHash() != disputedPayoutDistributionHash));
         require(getUniverse().getForkingMarket() != market);
         IReputationToken _reputationToken = getReputationToken();
         uint256 _amountToTransfer = _reputationToken.balanceOf(this);
+        if (bondRemainingToBePaidOut > bondAmount) {
+            uint256 _amountToCollectFeesOn = _amountToTransfer.min(bondRemainingToBePaidOut - bondAmount);
```

Comment:
`bondRemainingToBePaidOut - bondAmount` could be negative, which will underflow and result in a really big number.  I know we are doing a `min` here, but are we worried about this underflow?  If not, we should put a comment in explaining why its fine.

Reply:
This should be impossible due to the if condition it is in. I will add a comment though too

Reply:
Oh, oops.  Maybe I should sleep before reviewing this PR...

Reply:
Comment is probably not necessary, I should have seen that if statement.